pkg/constant/constants.go

```diff
@@ -4,4 +4,5 @@ const (
 	FinalizerName     string = "finalizer.lb.kubesphere.io/v1alpha1"
 	NodeFinalizerName string = "finalizer.lb.kubesphere.io"
 	IPAMFinalizerName string = "finalizer.ipam.kubesphere.io/v1alpha1"
+	EipRangeSeparator string = "~"
```

Comment:
Why not use`-` to separate? just like MetalLB

Reply:
OK, I'll change it.